pepper-apis/studybuilder-cli/studies/osteo/study-events.conf

```diff
@@ -1488,7 +1489,35 @@
       "maxOccurrencesPerUser": 1,
       "dispatchToHousekeeping": false,
       "order": 2
-    }
+    },
```

Comment:
Should there be changes to this file, or just new additions in `osteo-new-events.conf`?

Reply:
Those changes are not so important since the patch will reapply it. I'll revert this file to its original state.